app/models/notification.rb

```diff
@@ -105,7 +105,7 @@ def send_moderation_notification(notifiable)
       return unless notifiable.commentable
       return if UserBlock.blocking?(notifiable.commentable.user_id, notifiable.user_id)
 
-      Notifications::ModerationNotificationWorker.perform_async(notifiable.id)
+      Notifications::CreateRoundRobinModerationNotificationsWorker.perform_async(notifiable.id)
```

Comment:
I took a stab at refactoring our RoundRobinNotifications whilst keeping in mind some of the conventions that I recently wrote up as well. I tried to make things a little more clear, but I'm very open to feedback on this PR. If you think the code can be improved elsewhere as well, please let me know.